### ISSUE TITLE
[FEATURE] Update to skia milestone 90 (or greater)

### ISSUE THREAD

**Issue by Bluenuance** [mono/SkiaSharp#2230]
**Is your feature request related to a problem? Please describe.**

Usage of a simple shader within SKCanvasView leads to a crash.

Updating to MAUI from UWP is only possible for SKCanvasView. The usage of any shader for SKCanvasView is unfortunately blocked by this crash.

**Describe the solution you'd like**

Update to a skia milestone that has the SKVM enabled/integrated. (afaik 90+)

Update the Skia library to milestone 90+

**Additional context**

code that crashes within SKCanvasView:
```
var src = "half4 main(float2 coord) { return half4(1, 0, 0, 1); }";

using (var effect = SKRuntimeEffect.Create(src, out var errorText))
{	
	using (var shader = effect.ToShader(true, new SKRuntimeEffectUniforms(effect), new SKRuntimeEffectChildren(effect)))
	{
		var paint = new SKPaint
		{
		  Shader = shader
		};		

		canvas.DrawRect(new SKRect(0, 0, 100, 100), paint);
	}
}
```

**Comment by mgood7123** [comment]
> **Is your feature request related to a problem? Please describe.**
> 
> Usage of a simple shader within SKCanvasView leads to a crash.
> 
> Updating to MAUI from UWP is only possible for SKCanvasView. The usage of any shader for SKCanvasView is unfortunately blocked by this crash.
> 
> **Describe the solution you'd like**
> 
> Update to a skia milestone that has the SKVM enabled/integrated. (afaik 90+)
> 
> Update the Skia library to milestone 90+
> 
> **Additional context**
> 
> code that crashes within SKCanvasView:
> 
> ```
> var src = "half4 main(float2 coord) { return half4(1, 0, 0, 1); }";
> 
> using (var effect = SKRuntimeEffect.Create(src, out var errorText))
> {	
> 	using (var shader = effect.ToShader(true, new SKRuntimeEffectUniforms(effect), new SKRuntimeEffectChildren(effect)))
> 	{
> 		var paint = new SKPaint
> 		{
> 		  Shader = shader
> 		};		
> 
> 		canvas.DrawRect(new SKRect(0, 0, 100, 100), paint);
> 	}
> }
> ```

the shader crash is due to the BYTECODE_COMPILER not being defined when compiling skia

enabling it directly in the cpp files enables support for SKSL in CPU canvas's

see https://github.com/mono/SkiaSharp/issues/1822#issuecomment-1236014288

**Comment by cyraid** [comment]
An update of skia would be nice, then hopefully some of the shader functions would be included. 'mod' doesn't even seem to be included.

**Comment by dbriard** [comment]
Hi, is there a new SkiaSharp release planned with a fix for CPU shader? I come accros this issue and it is really problematic for the app I am working on.

**Comment by mattleibow** [comment]
As of today, `SKRuntimeEffect` is not supported on a raster/bitmap canvas and only on the GPU.

Duplicate of https://github.com/mono/SkiaSharp/issues/1822

**Comment by mattleibow** [comment]
Duplicate of https://github.com/mono/SkiaSharp/issues/1822
